Explorer/Assets/Scripts/ECS/SceneLifeCycle/SceneFacade/Systems/LoadSceneSystemLogic.cs

```diff
@@ -71,31 +100,14 @@ async UniTask<StreamableLoadingResult<string>> InnerFlow(SubIntention subIntenti
         ///     Loads scene metadata from a separate endpoint to ensure it contains "baseUrl" and overrides the existing metadata
         ///     with new one
         /// </summary>
-        private async UniTask<string> OverrideSceneMetadata(GetSceneFacadeIntention intention, string reportCategory, CancellationToken ct)
+        private async UniTask<UniTaskVoid> OverrideSceneMetadata(ISceneContent sceneContent, GetSceneFacadeIntention intention, string reportCategory, CancellationToken ct)
```

Comment:
I needed to return any type, otherwise I can't use it `WhenAll`:
```
            (SceneAssetBundleManifest manifest, _, ReadOnlyMemory<byte> mainCrdt) = await UniTask.WhenAll(loadAssetBundleManifest, loadSceneMetadata, loadMainCrdt);
```

It's just an API limitation

Reply:
In other cases I would use `UniTask`